lib/RequestTelemetry.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Stripe;
+
+/**
+ * Class RequestTelemetry
+ *
+ * Tracks client request telemetry
+ * @package Stripe
+ */
+class RequestTelemetry
+{
+    public $requestId;
+    public $requestDuration;
+
+    public function __construct($requestId, $requestDuration)
```

Comment:
missing `phpDoc`
`$requestId` probably must be a `string`
`$requestDuration` must be `int` in milliseconds